operators/tokenizer/bpetokenizer.hpp

```diff
@@ -20,6 +20,7 @@
 #include "nlohmann/json.hpp"
 #include "clip_tokenizer.hpp"
 #include "gpt2_tokenizer.hpp"
+#include "roberta_tokenizer.hpp"
```

Comment:
I cannot remember why these header files need to be included here? Can we remove them?

Reply:
We include the header files of the tokenizers that use bpe tokenization here so we can just include this file (bpetokenizer.hpp) for all bpe-based tokenizers. Unfortunately, we cannot remove them and instead just include bpetokenizer.hpp in the header files included here, such as roberta_tokenizer.hpp, because then when building we get 'struct' type redefinition errors for the customop primitives.